### PR TITLE
Fix relative links and general formatting

### PR DESCRIPTION
**Pull request by ScriptAutomate**
I'm creating automation tools that ingest and make use of these source files, but I ran into a lot of relative links that could be cleaned up due to strange or otherwise unnecessary formatting.

Applying these changes fixed everything for me.

- `WindowsServerDocs/troubleshoot/how-to-determine-the-minimum-staging-area-dfsr-needs-for-a-replicated-folder.md`: I wasn't sure what to do here. Perhaps it should be commented out with `<!-- -->` HTML commenting? Doesn't look like it is meant to be a published content block, but instead a TODO marker for a Microsoft employee? :stuck_out_tongue_closed_eyes: 
- `WindowsServerDocs/virtualization/hyper-v/manage/processor-compatibility-mode-hyper-v.md`: Converting the file format from UTF-8 w/ BOM to just UTF-8 (BOM causes problems in other editor tools, this is usually a Windows OS formatting issue)
- Rest is mostly path cleanup in links to images or other markdown files

### CODE REVIEW

**Comment by PRMerger3** [comment]
@ScriptAutomate : Thanks for your contribution! The author(s) have been notified to review your proposed change. @iainfoulds, @daniellee-msft

**Comment by PRMerger14** [comment]
@ScriptAutomate : Thanks for your contribution! The author(s) have been notified to review your proposed change. @iainfoulds, @daniellee-msft

**Comment by ScriptAutomate** [comment]
Hey! I opened this back in July, and I would really love this change in, otherwise I'm stuck rebasing and verifying after other merges make it in.

- @JasonGerend
- @nwashburn-ms
- @davannaw-msft

I'd love to contribute this PR and more in the future. If I need to follow a different process, please advise.

**Comment by jborsecnik** [comment]
@Xelu86, @robinharwood, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/troubleshoot/how-to-determine-the-minimum-staging-area-dfsr-needs-for-a-replicated-folder.md [hunk 1]
```diff
@@ -141,9 +141,8 @@ Based on this data you would set my staging area to 71 GB if you round up to the
 
 While a manual walkthrough is interesting it is likely not the best use of your time to do the math yourself. To automate the process, use command 3 from the examples above. The results will look like this
 
-> [![image]
-
-/// @Delan, these images (one thumbnail, one expanded) were pointing to non-existent blog sites. I was able to locate the full sized image from Wayback Machine and it is here - \\redmond\scratchfs\Teams\has\nedpyle\Delan\0876.image_03A39EFE.png .Can you add it to this article? ///
+> TODO
+> @Delan, these images (one thumbnail, one expanded) were pointing to non-existent blog sites. I was able to locate the full sized image from Wayback Machine and it is here - \\redmond\scratchfs\Teams\has\nedpyle\Delan\0876.image_03A39EFE.png .Can you add it to this article? ///
```
Comment (ScriptAutomate) [inline WindowsServerDocs/troubleshoot/how-to-determine-the-minimum-staging-area-dfsr-needs-for-a-replicated-folder.md:145]:
```suggestion
<!--
> TODO
> @Delan, these images (one thumbnail, one expanded) were pointing to non-existent blog sites. I was able to locate the full sized image from Wayback Machine and it is here - \\redmond\scratchfs\Teams\has\nedpyle\Delan\0876.image_03A39EFE.png .Can you add it to this article? ///
-->
```

**Comment by robinharwood** [comment]
Thanks for you contribution @ScriptAutomate, I'm reviewing these now.

**Comment by robinharwood** [comment]
Hi @ScriptAutomate - I've made a few minor changes but you should this merged by Monday.

Thanks,

Robin

**Comment by ScriptAutomate** [comment]
@robinharwood Thanks so much for your review, approval, and merging!!

**Pull request merged**